components/myforms/Card/Card.tsx

```diff
@@ -15,13 +15,6 @@ export const Card = (props: CardProps): React.ReactElement => {
   const { id, titleEn, titleFr, url, date, isPublished } = props;
   const { t, i18n } = useTranslation(["my-forms", "common"]);
 
```

Comment:
This was a little bit of creativity. On second thought, best to drop and stay with standard way of adding semantics for now.